Source/JavaScriptCore/dfg/DFGByteCodeParser.cpp

```diff
@@ -4375,6 +4375,7 @@ bool ByteCodeParser::handleIntrinsicGetter(Operand result, SpeculatedType predic
         return false;
 
     switch (variant.intrinsic()) {
+    case DataViewByteLengthIntrinsic:
```

Comment:
For `DataViewByteLengthIntrinsic` case, you can emit `CheckDetached` to ensure that DataView is not detached. (And then you can also ensure that the array is not detached in code generation).
And you can stop inlining here when we had `BadIndexingType` OSR exit kind here via `hasExitSite`.